test/e2e-tests.sh

```diff
@@ -172,7 +172,7 @@ function kafka_setup() {
   sed 's/namespace: .*/namespace: kafka/' ${STRIMZI_INSTALLATION_CONFIG_TEMPLATE} > ${STRIMZI_INSTALLATION_CONFIG}
   kubectl apply -f "${STRIMZI_INSTALLATION_CONFIG}" -n kafka
   kubectl apply -f ${KAFKA_INSTALLATION_CONFIG} -n kafka
-  kubectl apply -f ${KAFKA_TOPIC_INSTALLATION_CONFIG} -n kafka
+  # kubectl apply -f ${KAFKA_TOPIC_INSTALLATION_CONFIG} -n kafka
```

Comment:
left as a comment?

Reply:
It's deliberate: it's not needed anymore but i wish to leave it for anybody who needs a quick test topic

Reply:
I'm not sure somebody will be looking at knative e2e testing for basic strimzi configuration setup?